packages/react-components/react-avatar/src/components/AvatarGroupItem/useAvatarGroupItemStyles.ts

```diff
@@ -150,16 +155,57 @@ const useSpreadStyles = makeStyles({
   xl: { '&:not(:first-child)': { marginLeft: tokens.spacingHorizontalXL } },
 });
 
+/**
+ * Styles for the pie layout
+ */
+const usePieStyles = makeStyles({
+  base: {
+    position: 'absolute',
+  },
+  slices: {
+    // Two slices
+    '&:nth-of-type(1):nth-last-of-type(2)': {
```

Comment:
It might be nice to add comments for these selectors, like `// 1st of 2 items`, `// 2nd of 3 items`, etc.